xml/hwbp_registers.xml

```diff
@@ -1252,11 +1252,13 @@ same project unless stated otherwise.
 
         This register is accessible as {csr-tdata1} when {tdata1-type} is 5.
 
-        This trigger may fire on up to XLEN of the Exception Codes defined in
-        `mcause` (described in the Privileged Spec, with Interrupt=0). Those
-        causes are configured by writing the corresponding bit in {csr-tdata2}.
-        (E.g.  to trap on an illegal instruction, the debugger sets bit 2 in
-        {csr-tdata2}.)
+        This trigger can match when a trap is taken due to an exception.
+
+        The trigger may be configured to match on up to XLEN of the Exception
+        Codes defined in `mcause` (described in the Privileged Spec, with
+        Interrupt=0). Those causes are configured by writing the corresponding
+        bit in {csr-tdata2}. (e.g. to have etrigger match on an illegal
+        instruction trap, the debugger sets bit 2 in {csr-tdata2}.)
```

Comment:
This still doesn't say that the trigger matches when an exception is taken, which is what your change description says you wanted to accomplish.